lib/partition_setup.pm

```diff
@@ -486,9 +486,10 @@ sub select_first_hard_disk {
               if ($matched_needle && $matched_needle->{needle}->has_tag("hard-disk-dev-$device-selected"));
         }
     }
-    # Check if sda is still/already selected, if not select it
-    assert_screen [qw(select-hard-disks-one-selected hard-disk-dev-sda-not-selected)];
+    # Check if sda is still/already selected, if not select it, if none disk selected, select the first one
```

Comment:
Suggestion:
Select the first disk device if none selected

Reply:
Updated